src/Button/Button.module.scss

```diff
@@ -141,3 +141,17 @@
     background: var(--athens) !important;
   }
 }
+
+.loading {
+  pointer-events: none;
+  color: transparent;
+  fill: transparent;
+  position: relative;
+}
+
+.loader {
+  position: absolute;
+  top: 14px;
```

Comment:
`Button` may have different height, so the hardcoded value does not handle all cases. `Button` itself is `flex/inline-flex` container which [has an `align-items: center` property](https://github.com/santiment/san-ui/pull/78/files#diff-a1b683f5b5492bb0267a42765ab8cb8cR11), so it will align loader automatically